.github/workflows/pre-merge.yml

```diff
@@ -143,7 +143,13 @@ jobs:
       - name: Sanitize project folder
         id: sanitize
         run: |
-          SANITIZED_PROJECT_FOLDER=$(echo "${{ inputs.project_folder }}" | tr '/' '-')
+          # check if inputs.project_folder is set, if not return repository name
+          if [ -z "${{ inputs.project_folder }}" ] || [ "${{ inputs.project_folder }}" = "." ]; then
+            SANITIZED_PROJECT_FOLDER="${GITHUB_REPOSITORY#"${GITHUB_REPOSITORY_OWNER}/"}"
+          else
+            SANITIZED_PROJECT_FOLDER=$(echo "${{ inputs.project_folder }}" | tr '/' '-')
+          fi
```

Comment:
This would change content of `REPO_PROJECT` (few lines later).

Reply:
that's the intention. when it's empty or . the reports look like gitleask-..json